### PR TITLE
Wrapping Pardiso

### PR DESCRIPTION
**Pull request by sgkang**
Designed for handling real symmetric and complex symmetric matrix. 

Requires:

https://github.com/dwfmarchant/pyMKL

### CODE REVIEW

**Comment by rowanc1** [comment]
Could you copy the function you need out of SimPEG to keep the requirements down?

File: pymatsolver/Mumps/Makefile.seogi [hunk 1]
```diff
@@ -0,0 +1,25 @@
+INCMUMPS  = -I/usr/local/Cellar/mumps/5.0.0_2/include -L/usr/local/Cellar/mumps/5.0.0_2/lib -L/usr/local/opt/scotch5/lib -I/usr/local/opt/scotch5/include -I/usr/local/Cellar/open-mpi/1.8.4/include -L/usr/local/opt/libevent/lib -L/usr/local/Cellar/open-mpi/1.8.4/lib -lmpi
```
Comment (rowanc1) [inline pymatsolver/Mumps/Makefile.seogi]: Can you please delete this file?

**Comment by sgkang** [comment]
Not sure sp is not recognized ... any idea @lheagy @rowanc1

File: pymatsolver/Mumps/Makefile.seogi (hunk 1 above)
Comment (sgkang) [inline pymatsolver/Mumps/Makefile.seogi]: Yep. Will do

**Comment by rowanc1** [comment]
See #8

**Pull request closed without merging**